CHANGELOG.md

```diff
@@ -1,3 +1,67 @@
+# dbt_shopify_source v0.14.0
+[PR #93](https://github.com/fivetran/dbt_shopify_source/pull/93) includes the following changes:
+
+## Breaking Changes
+- Updated for connector changes released on January 6, 2025. See the [release notes](https://fivetran.com/docs/connectors/applications/shopify/changelog#january2025) for more details. Added the following columns to the `stg_shopify__*` staging tables:
+  - `inventory_item`:
```

Comment:
looks like we added `duplicate_sku_count` for `inventory_item` as well, can we add that in?

Reply:
Good catch. Added.